### DISCUSSION TITLE
Obj & glTF native support

### DISCUSSION THREAD

**Discussion by DelinWorks** [axmolengine/axmol#591] [Ideas]
Do you plan to add .obj and .gltf support for 3D models file formats rather than using c3b or c3t to make workflow easier?

**Comment by halx99** [comment]
The .obj already supported: https://github.com/adxeproject/adxe/blob/dev/core/3d/CCSprite3D.cpp#L251

**Reply by DelinWorks** [reply]
cool! havent found any docs about it, what about gltf?

**Reply by halx99** [reply]
No plan for support gltf yet
